docs/1-Getting-Started/2-Starting-new-project.md

```diff
@@ -18,164 +12,164 @@ Also, make sure you're running the latest version of **kool**. Run the following
 $ kool self-update
 ```
 
-### 1. Run `kool create`
+### 1. Create a New Project
 
-Use the `kool create <preset> <project-name>` command to create your new project.
+Create a working directory for your new project, and move into it.
 
 ```bash
-$ kool create laravel my-project
+$ mkdir my-app
+$ cd my-app
 ```
 
-Under the hood, this command will run `composer create-project --no-install --no-scripts --prefer-dist laravel/laravel <project-name>` (using a customized **kool** Docker image: <a href="https://github.com/kool-dev/docker-php" target="_blank">kooldev/php:7.4</a>).
+### 2. Add docker-compose.yml
 
-After installing Laravel, `kool create` automatically runs the `kool preset laravel` command, which helps you easily set up the initial tech stack for your project using an interactive wizard.
+Create a **docker-compose.yml** file in the root of your working directory, and then copy/paste one of the YAML templates provided below into **docker-compose.yml**.
```

Comment:
I don't like at first the optional basic/complex example mixup. I would stick to the simple version only, that being the goal here. I would like nevertheless even another call to action to Presets and their more complex setup.

Reply:
@fabriciojs - moved the 2nd docker-compose.yml template to bottom of page under "Diving Deeper".